config/urlProvider.ts

```diff
@@ -3,4 +3,6 @@ export const BASE = "https://traineeautomation.azurewebsites.net/";
 export const URLS = {
   ADDUSER: "/Forms/AddUser",
   ADDADDRESS: "/Forms/AddAddress",
+  EDITUSER: "/Forms/EditUser",
```

Comment:
It's better to give names for constants in this way: 
using words in uppercase and use '_' as a separator between words: 
EDIT_USER
DELETE_USER
etc